tf-psa-crypto/drivers/builtin/src/psa_crypto_ecp.c

```diff
@@ -679,8 +679,35 @@ psa_status_t mbedtls_psa_ecp_export_public_key_iop_setup(
     return status;
 }
 
+psa_status_t mbedtls_psa_ecp_export_public_key_iop_complete(
+    mbedtls_psa_export_public_key_iop_t *operation,
+    uint8_t *pub_key,
+    size_t pub_key_size,
+    size_t *pub_key_len)
+{
+    int status = 0;
+
+    if (mbedtls_ecp_is_zero(&operation->key->Q)) {
+        mbedtls_psa_interruptible_set_max_ops(psa_interruptible_get_max_ops());
+
+        status = mbedtls_ecp_mul_restartable(&operation->key->grp, &operation->key->Q,
+                                             &operation->key->d, &operation->key->grp.G,
+                                             mbedtls_psa_get_random, MBEDTLS_PSA_RANDOM_STATE,
+                                             &operation->restart_ctx);
+        operation->num_ops += operation->restart_ctx.ops_done;
+    }
+
+    if (status == 0) {
```

Comment:
`status` is the return of `mbedtls_ecp_mul_restartable()` which return 0 on success and not PSA error codes I know PSA_SUCCESS happen to be 0 as well but I prefer checking on the same error codes that the function returns (in case PSA_SUCCESS changes or any other reason)